### PR TITLE
add NNLL reweighting for Z peak

### PR DESCRIPTION
**Pull request by alesaggio**

### CODE REVIEW

File: interface/Types.h [hunk 1]
```diff
@@ -83,6 +83,7 @@ namespace HtoZA {
         float trigger_efficiency;
         float trigger_efficiency_downVariated;
         float trigger_efficiency_upVariated;
+        float ptllDYW_NNLL;
```
Comment (OlivierBondu) [inline interface/Types.h]: could you specify it's NLO->NNLL ? We did not add the LO -> NNLL weights

File: plugins/HtoZAAnalyzer.cc [hunk 2]
```diff
@@ -464,6 +462,16 @@ void HtoZAAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, con
                 continue;
             }
 
+            // Store NNLL reweighting for Z peak -->
+            // https://github.com/latinos/PlotsConfigurations/blob/master/Configurations/ggH/Full2016/samples.py#L173
```
Comment (OlivierBondu) [inline plugins/HtoZAAnalyzer.cc]: Could you replace by the permanent link instead ? (https://github.com/latinos/PlotsConfigurations/blob/a7f853e7badc8be2c84c4830ed410ec82025cf57/Configurations/ggH/Full2016/samples.py#L173)

**Review by OlivierBondu — CHANGES_REQUESTED** [review]
Thanks ! only minor comments :smile: 

also: maybe we should profit from the occasion to put the LO->NNLL weights as well ? we don't know which surprise requests the review will hold....

**Comment by alesaggio** [comment]
Updated! :)

**Review by OlivierBondu — APPROVED** [review]

**Comment by OlivierBondu** [comment]
Thanks!

**Pull request merged**